index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="stylesheet" href="main.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="navigator">
+        <h2>Welcome</h2>
+        <img src="images/more.png" alt="more" />
+      </nav>
+    </header>
+    <main>
+      <div class="main-title">
+        <div>
+          <h2 class="main-header">
+            Hey there. I'm Nati
+            <span class="highligh-title">I'm a software developer</span>
+          </h2>
+        </div>
+        <div>
+          <p class="about-me">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+        </div>
+      </div>
+      <div class="main-icons">
```

Comment:
- [ ] [REQUIRED] I kindly suggest that you group your list of icons in a `ul` tag and use a `li` tag per icon